parser-typechecker/src/Unison/Codebase/SqliteCodebase/Migrations.hs

```diff
@@ -51,6 +53,30 @@ migrations getDeclType termBuffer declBuffer rootCodebasePath =
       (7, migrateSchema6To7)
     ]
 
+data CodebaseVersionStatus
+  = CodebaseUpToDate
+  | CodebaseUnknownSchemaVersion SchemaVersion
+  | CodebaseRequiresMigration
+      -- Current version
+      SchemaVersion
+      -- Required version
+      SchemaVersion
+  deriving stock (Eq, Ord, Show)
+
+checkCodebaseIsUpToDate ::
+  MonadUnliftIO m =>
+  Sqlite.Connection ->
+  m CodebaseVersionStatus
+checkCodebaseIsUpToDate conn = do
+  Sqlite.runWriteTransaction conn \run -> do
```

Comment:
Could we change this to `runTransaction`?

Reply:
(or change the type to `Transaction CodebaseVersionStatus` and `runTransaction` outside this function)